packages/sanity/src/core/store/key-value/storage/serverStorage.ts

```diff
@@ -26,7 +26,6 @@ export function createServerStorage({client: _client}: ServerStorageOptions): Se
     const value = await client
       .request<KeyValuePair[]>({
         uri: `/users/me/keyvalue/${keys.join(',')}`,
-        withCredentials: true,
```

Comment:
@cngonzalez  Can you confirm this to be safe? Any idea why we passed `withCredentials: true` here explicitly? (this should have implicitly been the case unless a client not based on the source's `getClient`/`useClient` was used)